lib/seattleflu/db/cli/command/longitudinal.py

```diff
@@ -0,0 +1,278 @@
+"""
+Parse and upload longitudinal data.
+
+Longitudinal childcare data will contain PII (personally identifiable
+information) and unnecessary information that does not need to be stored. This
+process will only pull out specific columns of interest that will then be stored
+in the receiving schema of ID3C.
+"""
+import re
+import click
+import logging
+import numpy as np
+import pandas as pd
+import seattleflu.db as db
+from math import ceil
+from seattleflu.db.session import DatabaseSession
+from seattleflu.db.cli import cli
+from . import LOG, add_metadata, generate_hash, barcode_quality_control
+
+
+@cli.group("longitudinal", help = __doc__)
+def longitudinal():
+    pass
+
+@longitudinal.command("parse")
+@click.argument("baseline_filename",
+    metavar = "<Longitudinal Childcare Data baseline (enrollments) filename>")
+@click.argument("weekly_filename",
+    metavar = "<Longitudinal Childcare Data weekly assessments filename>")
+@click.option("-o", "--output",
+    metavar="<output filename>",
+    help="The filename for the output of missing barcodes")
+
+def parse(baseline_filename, weekly_filename, output):
+    """
+    Process and insert longitudinal childcare data from SCH.
+
+    All childcare records parsed are output to stdout as newline-delimited JSON
+    records.  You will likely want to redirect stdout to a file.
+    """
+    baseline_records, weekly_records = load_data(baseline_filename, weekly_filename)
+
+    lnginal_records = baseline_records.merge(weekly_records, how='left',
+                                             on='study_id',
+                                             suffixes=['', '_followup'])
+
+    lnginal_records = weekly_assessments_wide_to_long(lnginal_records)
+    lnginal_records = select_all_that_apply_wide_to_long(lnginal_records)
+    lnginal_records = duplicate_audere_keys(lnginal_records)
+    lnginal_records = insert_static_values(lnginal_records)
+    lnginal_records = hash_pii(lnginal_records)
+
+    barcode_quality_control(lnginal_records, output)
+
+    print(lnginal_records.to_json(orient='records', lines=True))
+
```

Comment:
I'll refactor this to use the new `dump_ndjson` function Thomas recently wrote for clinical records